GitAutomation/GitAutomation.psd1

```diff
@@ -148,8 +148,12 @@
             # ReleaseNotes of this module
 
             ReleaseNotes = @'
-* Added `Get-GitConfiguration` function for getting Git configuration values.
-* Added `Remove-GitConfiguration` function for removing/unsetting Git configuration values.
+* Fixed: `ConvertTo-GitFullPath` doesn't properly encode the returned Uri string. (thanks @thorbenw)
+  * as described in Issue #32

```

Comment:
Remove this line, ` * as described in Issue #32`, this is specific to a GitHub issue and won't link anywhere when these release notes are displayed on the PowerShell Gallery or elsewhere. The PR already links to the issue.

Reply:
Done.